tests/common/plugins/sanity_check/constants.py

```diff
@@ -22,43 +22,43 @@
 # Recover related definitions
 RECOVER_METHODS = {
     "config_reload": {
-        "cmd": "bash -c 'config reload -y &>/dev/null'",
-        "reboot": False,
-        "adaptive": False,
-        'recover_wait': 120
-    },
-    "config_reload_f": {
```

Comment:
Why removed `config_reload_f`?

Reply:
config_reload function has include `config reload -f` logic implicitly

Reply:
https://github.com/sonic-net/sonic-mgmt/blob/b10f9b6c7d345581816ee30dcd457a365a8f2e49/tests/common/config_reload.py#L115C12-L115C12

Reply:
OK. Is there any code using this `config_reload_f`? If yes, removing it may cause issue without updating the calling code.

Reply:
> OK. Is there any code using this `config_reload_f`? If yes, removing it may cause issue without updating the calling code.

No, I have checked that, recover is the only place that uses config_reload_f.